srcCpp/MessagingIF.h

```diff
@@ -184,6 +184,18 @@ class IMessaging
     virtual bool supports_listener() = 0;
     virtual bool supports_discovery() = 0;
 
+    /*
+     * @brief This function calculate the overhead bytes that all the
```

Comment:
/*
     * @brief This function calculates the overhead bytes added by all the
     * members on the TestData_* type, excluding the content of the sequence.